### PR TITLE
docs: docker usage

### PR DESCRIPTION
**Pull request by developStorm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,25 @@ ZGrab is a fast, modular application-layer network scanner designed for completi
...
+echo 'example.com' | docker run --rm -i ghcr.io/zmap/zgrab2 http
+```
+
+For more complex scanning scenarios, such as using multiple modules or custom configurations, you can create a configuration file and pass it to the container:
+
+```shell
+docker run --rm -i -v /path/to/your/config.ini:/config.ini ghcr.io/zmap/zgrab2 multiple -c /config.ini
+```
+
+Replace `/path/to/your/config.ini` with the path to your configuration file on the host machine. See follow for more details on configurations.
```
Comment (phillip-stephens) [inline README.md:24]: Can we link to the section on Multiple Module Usage?
Something like
"See Multiple Module Usage for more details on configurations."

Where "Multiple Module Usage" is a link to that section of the README

**Review by phillip-stephens — CHANGES_REQUESTED** [review]
LGTM overall, tested the docker image locally and looks good!

Minor nits on verbage, see comment!

File: README.md (hunk 1 above)
Comment (developStorm) [reply README.md]: 8d2bf0422fe1dd9fc2639270da2c78c7ec9365de

**Review by phillip-stephens — APPROVED** [review]
LGTM, thanks @developStorm

**Pull request merged**